src/main/webapp/app/view/project/admin/users/project-user-modal.component.ts

```diff
@@ -0,0 +1,35 @@
+import { AfterViewInit, Component, OnDestroy } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+import { NgbModal, NgbModalRef } from '@ng-bootstrap/ng-bootstrap';
+import { JhiEventManager } from 'ng-jhipster';
+import { ProjectUsersInviteComponent } from 'app/view/project/admin/users/project-users-invite.component';
+
+@Component({
+  template: '',
+})
+export class ProjectUserModalComponent implements AfterViewInit, OnDestroy {
+  private modalRef?: NgbModalRef;
+
+  constructor(private modalService: NgbModal, private eventManager: JhiEventManager, private activatedRoute: ActivatedRoute) {}
+
+  ngAfterViewInit(): void {
+    this.activatedRoute.data.subscribe(({ project }) => {
+      this.modalRef = this.modalService.open(ProjectUsersInviteComponent, {
+        beforeDismiss(): boolean {
+          window.history.back();
+          return true;
+        },
+      });
+      (this.modalRef.componentInstance as ProjectUsersInviteComponent).setProject(project);
+    });
+  }
+
+  ngOnDestroy(): void {
+    if (this.modalRef) {
+      this.modalRef.close();
+    }
+
+    this.eventManager.broadcast('invitationListModification');
```

Comment:
the broadcast should only happen on "save", not on every close (e.g. just exit as "cancel"). move to the component itself (I did this in #89).

Reply:
done